XcodePostBuild.cs

```diff
@@ -271,6 +271,12 @@ private static void PatchUnityNativeCode(string pathToBuiltProject)
         {
             EditSplashScreenMM(Path.Combine(pathToBuiltProject, "Classes/UI/SplashScreen.mm"));
         }
+        
+        // TODO: There are possibly more versions with this issue.
+        if (Application.unityVersion.StartsWith("2018."))
```

Comment:
I wonder if you simply put this condition into the if condition above, will it still work?

Looking at the function in the issue you mentioned, the logic is pretty similar with 2017.3+ versions.  If `hasStoryboard` is set to false, these storyboards won't conflict with each other, right?

Reply:
I'll give it a try this week and report back shortly!